.vale/styles/Konflux/DirectAddress.yml

```diff
@@ -0,0 +1,9 @@
+extends: substitution
+message: "Use 'you' instead of '%s' for direct address"
```

Comment:
This error message looks sus
```
modules/troubleshooting/pages/builds.adoc:40:88:Konflux.DirectAddress:Use 'you' instead of 'you' for direct address
```